Makefile

```diff
@@ -26,6 +26,8 @@ deps.darwin.amd64:
 
 	git clone -b v${DUCKDB_VERSION} --depth 1 https://github.com/duckdb/duckdb.git
 	cd duckdb && \
+	git fetch origin main && \
+	git cherry-pick -n -m 1 cafbcfb3c66634e6c5d7e9b8eb692791b93139d1 && \
```

Comment:
Would recommend increasing the generic-ness a bit here (since someone else might be doing DuckDB upgrades next time and miss this).

Maybe something like:
- Add a `DUCKDB_CHERRYPICK_COMMIT=` on line 2 in this file, and reference it here as `${DUCKDB_CHERRYPICK_COMMIT}` (with a small comment about how to use it)
- Use a bash if check to only run these commands if `DUCKDB_CHERRYPICK_COMMIT` is not the empty string
-